### PR TITLE
[Intel Mkl] Adding Bazel version and BFloat16 support to the MKL CI

### PR DESCRIPTION
**Pull request by claynerobison**
MKL CI containers need to support different Bazel versions and BFloat16.

### CODE REVIEW

**Comment by claynerobison** [comment]
Hi @penpornk and @rthadur. Thanks for reviewing this. We have more updates we want to push to these files, but would like your approval here first.

**Review by penpornk — CHANGES_REQUESTED** [review]
I'm so sorry for the delay! I have a few minor comments.

File: tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh [hunk 1]
```diff
@@ -103,11 +107,21 @@ function build_container()
     TF_DOCKER_BUILD_ARGS+=("--build-arg CONFIG_V2_DISABLE=--disable-v2")
   fi
 
+  #Add build arg for bfloat16 build
+  if [[ \${BUILD_TF_BFLOAT16_CONTAINERS} == "yes" ]]; then
```
[the remark this answers is not in the document]
Comment (claynerobison) [reply tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh]: You are correct! Good catch.

**Review by penpornk — APPROVED** [review]
Thank you again for the PR!

**Pull request merged**